### DISCUSSION TITLE
Can we disable preset/bookmark editing for a specific role?

### DISCUSSION THREAD

**Discussion by ThijsCatwise** [directus/directus#24066] [Feature Requests]
### Summary

When I'm making a CMS for a customer I don't want them to change the presets. It needs to be one global preset to all users which can be set by an admin.

### Motivation

Because customers always seem to find a way to f* things up haha!

### Detailed Design

Just the current design. But then readonly.

### Requirements List

None, because I think all the above is clear.

### Drawbacks

Static feeling for clients

### Alternatives

-

### Adoption Strategy

-

**Comment by burka** [comment]
We have the same problem from time to time, some users just change their presets and wonder why it looks different. 

Having a _reset to default_ option would help too :-)

**Reply by rijkvanzanten** [reply]
I'm also thinking through a world where by default the preset is shared between all users, so whatever's the last state is there for everybody, with a "make a copy of this for just me". That way there's less confusion around what somebody on the other end sees. Needs a bit more thinking through though 🙂 🧠

**Reply by Ayo91** [reply]
> We have the same problem from time to time, some users just change their presets and wonder why it looks different.
> 
> Having a _reset to default_ option would help too :-)

I have implemented a reset to default option for the collections.  I can propose it as a pull request integration. Example:

![image](https://github.com/user-attachments/assets/7e056871-aad1-4629-b5e4-b0f57e27b1a6)

Feature request: https://github.com/directus/directus/discussions/24499

@rijkvanzanten
